pypit/arutils.py

```diff
@@ -69,12 +70,18 @@ def bspline_fit(x,y,order=3,knots=None,everyn=20,xmin=None,xmax=None,w=None,bksp
       Knot everyn good pixels, if used
     bkspace: float 
       Spacing of breakpoints in units of x
+    bspline_par : dict, optional
+      Convenient way to set several parameters (like settings)
 
     Returns:
     ---------
     tck : tuple
       describes the bspline
     '''
+    # Unpack bspline_par
+    if bspline_par is not None:
+        for key,value in bspline_par.items():
+            key = value
```

Comment:
doesn't this just overwrite key with each element in the loop? We might need to do something clever here like:

`eval("{0:s}={1:s}".format(key, value))`

Reply:
Agreed.  In the end, I decided to keep a cheat
in func_fit:

```
        if bspline_par is None:
            bspline_par = {}
        # TODO -- Deal with this kwargs-like kludge
        return bspline_fit(x, y, order=deg, w=w, **bspline_par)
```

Am pretty sure we are headed toward using ```bspline_iterfit```
anyhow.